### ISSUE TITLE
allow wrapping of GRPC client

### ISSUE THREAD

**Issue by willscott** [drand/drand#550]
```
Jun 12 16:58:00 relay-http-i-002237bcee5f4f055.dev1.drand drand-relay-http ts="Fri, 12 Jun 2020 16:58:00 UTC" call=poll.go:52 level=error polling_client="failed to watch" err="rpc error: code = Unknown desc = can't retrieve beacon: beacon not found in database <nil>"
```

hypothesis: asking for a round in the future

**Comment by willscott** [comment]
something to do with
> This also allows gRPC only client to inherit the goodness that client.Wrap adds.
in #548

**Comment by alanshaw** [comment]
😂 "goodness"

**Comment by willscott** [comment]
appears fixed with #552
